src/main/java/com/openshift/internal/restclient/model/Pod.java

```diff
@@ -149,4 +149,17 @@ public IContainer addContainer(String name) {
         return Collections.emptyList();
     }
 
+    @Override
+    public boolean isReady() {
+        ModelNode node = get(POD_STATUS_CONTAINER_STATUSES);
+        if (node.getType() != ModelType.LIST) {
+            return false;
+        }
+        boolean allContainersReady = true;
+        for (ModelNode containerStatus : node.asList()) {
```

Comment:
Hi! I was wondering if you would find useful to provide a `getContainerStatuses()` method on the Pod class that would return a `List<IContainerStatus>` and iterate over them calling `IContainerStatus` `isReady()`, `getName()`, etc...
This way you can implement this functionality and also make the user able to query the container statuses if it is found useful.

Reply:
@rbocanegra excellent idea! Want to PR this?

Reply:
Sure, I would need a couple of weekends. But I will try to make a PR.

Reply:
@rbocanegra great, looking forward to it 👍

Reply:
@rbocanegra filed your suggested improvement to #379